manual/manual/cmds/intf-c.etex

```diff
@@ -1597,9 +1603,27 @@ With "ocamlopt", you should do:
 % -output-obj" already contains the OCaml runtime library as
 % well as all the needed C libraries.
 
+For the final linking phase, in addition to the object file produced
+by "-output-complete-obj", you will have only to provide all C
+libraries that are required by the OCaml runtimes used.
+
+For instance, assume the OCaml part of your program uses the
+Unix library.  With "ocamlc", you should do:
+\begin{alltt}
+        ocamlc -output-complete-obj -o camlcode.o unix.cma {\it{other}} .cmo {\it{and}} .cma {\it{files}}
+        cc -o myprog {\it{C objects and libraries}} \char92
```

Comment:
I don't find that particularly intelligible and rather ambiguous, maybe `camlcode.o + -lc -lm` ? In particular it seems to me that should not repeat the autolink libraries here.

Reply:
I was just suggesting examples of what you may need to add here. It's just that it should be made clear that you should not add the autolink libs otherwise you will be subject to double linking. I just find that not very clear in the current text.

(Aside. For this reason I personally find the behaviour of `-output-complete-obj` dubious, you may have other C object files at that point that need the C libs that you autolinked via -output-complete-obj).

Reply:
(In your case you are not the target audience for `-output-complete-obj`.)